### ISSUE TITLE
Add new property owner 'UserProperties' that stores application state

### ISSUE THREAD

**Issue by alexanderbock** [OpenSpace/OpenSpace#716]
With the ability to add and remove properties via Lua

**Comment by emiax** [comment]
A sketch of how it might work:

```
openspace.addCustomProperty({
    Identifier = "TargeteEarthSize",
    Type = "FloatProperty",
    Description = "Target Earth Scale",
})

openspace.addCustomProperty({
    Identifier = "IncreasePlanetSize",
    Type = "BooleanProperty",
    Description = "Increase Earth Size",
    OnChange = "function (value) if (value) then openspace.setPropertyValue('Scene.Earth.Scale.Scale', openspace.getPropertyValue('CustomProperties.TargeteEarthSize')) else openspace.setPropertyValue('Scene.Earth.Scale.Scale', 1) end"
})

openspace.setPropertyValue("CustomProperties.IncreasePlanetSize", true)
```

**Comment by alexanderbock** [comment]
Example for usage:   magnetogram_textures of @ElonOlsson  branch

**Comment by alexanderbock** [comment]
Implemented in ee735618b480e3939db455d28ecf712332508730
